dev/io.openliberty.io.smallrye.config/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -20,31 +20,64 @@ org.eclipse.jdt.core.compiler.debug.sourceFile=generate
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
 org.eclipse.jdt.core.compiler.source=1.8
+org.eclipse.jdt.core.formatter.align_assignment_statements_on_columns=false
```

Comment:
did you intend to checkin changes to this file?

Reply:
Yes, I made changes to the formatter settings to allow it to wrap between chained calls (e.g. `a.getB().getC()`) and between attributes in an annotation.

Without this I was getting bad wrapping decisions like this one where the formatter was happy to break the line in the middle of the array value for the `service` attribute, but not between each attribute.
```
@Component(service = { ConfigProviderResolver.class,
                       ApplicationStateListener.class }, configurationPolicy = ConfigurationPolicy.IGNORE, property = { "service.vendor=IBM" }, immediate = true) 
```

The other changes in this file are a result of the newer version of eclipse adding values for options which didn't exist when the file was last written.

However, the changes to the other settings file look less good. It's added new templates for Java 9 features which includes things like `@author` tags which we don't use in OL.